packages/stark-ui/karma.conf.js

```diff
@@ -18,6 +18,7 @@ const karmaTypescriptBundlerAlias = {
 	"@ng-idle/keepalive": "../stark-core/node_modules/@ng-idle/keepalive/bundles/keepalive.umd.js",
 	"@ngrx/store": "../stark-core/node_modules/@ngrx/store/bundles/store.umd.js",
 	"@ngrx/effects": "../stark-core/node_modules/@ngrx/effects/bundles/effects.umd.js",
+	"@ngrx/effects/testing": "../stark-core/node_modules/@ngrx/effects/bundles/effects-testing.umd.js",
```

Comment:
Is this related to the issue with Prettier and Karma or is it just a small fix at the same time ? 😊

Reply:
It is just an addition (not even a fix) cause I noticed the file is  imported correctly but I wanted to explicitly set it here as we do with the rest of imports just to be safe 😉

And no, is not related with the karma issue but I took the chance to add it...